bolt/lib/Profile/YAMLProfileReader.cpp

```diff
@@ -354,7 +357,15 @@ Error YAMLProfileReader::readProfile(BinaryContext &BC) {
       matchProfileToFunction(YamlBF, Function);
   }
 
-  for (auto &[CommonName, LTOProfiles]: LTOCommonNameMap) {
```

Comment:
Could you please try to build with:
```
    for (const auto &[CommonName, LTOProfiles]: LTOCommonNameMap) {
```
? I don't have a Windows builder available to test.

Reply:
It has been confirmed that the new code compiles properly on Windows.